cylc/flow/scheduler_cli.py

```diff
@@ -561,10 +561,14 @@ def _upgrade_database(db_file: Path) -> None:
 
 
 def _print_startup_message(options):
-    """Print the Cylc header including the CLI logo."""
+    """Print the Cylc header including the CLI logo to the user's terminal."""
     if (
         cylc.flow.flags.verbosity > -1
         and (options.no_detach or options.format == 'plain')
+        # don't print the startup message on reinvocation (note
```

Comment:
Are you happy to skip the Logo if I set `--localhost` manually? I'm fine with it.

Reply:
Yes, this is a caveat of the hacky approach, however, it avoids the need for another CLI argument and there is very little use case for specifying localhost.